src/node-resolver.ts

```diff
@@ -20,22 +19,19 @@ type NodeResolverOptions = {
    * @default ['default', 'module', 'import', 'require']
    */
   conditionNames?: string[]
-} & Omit<ResolveOptions, 'useSyncFileSystemCalls'>
+} & ResolveOptions
 
 export function createNodeResolver({
   extensions = ['.mjs', '.cjs', '.js', '.json', '.node'],
   conditionNames = ['default', 'module', 'import', 'require'],
```

Comment:
https://nodejs.org/dist/latest-v14.x/docs/api/packages.html#packages_conditional_exports

https://github.com/wooorm/import-meta-resolve/blob/fa79719d4de9add0743572f7248d9c3efafa35d0/lib/utils.js#L12

`default` should be last applied?

Reply:
> `default` should be last applied?

Hmmm, yeah it should be the last.